packages/aws-cdk-lib/aws-codepipeline/README.md

```diff
@@ -479,3 +527,41 @@ const target = new chatbot.SlackChannelConfiguration(this, 'MySlackChannel', {
 declare const pipeline: codepipeline.Pipeline;
 const rule = pipeline.notifyOnExecutionStateChange('NotifyOnExecutionStateChange', target);
 ```
+
+## Trigger
```

Comment:
Separate feature, separate PR imo. Unless you can convince me that these are somehow intertwined. Think about if we have to revert, we don't want to have to revert everything just because they were bundled together in one PR

Reply:
I think we need to see the cdk logic in the same way of pipeline features, 

V2 is actually V1 plus variables and tag triggers, 

Maybe extending V1 to cover v2 can be elegant instead of changing V1 

Also by just adding pipeline type v2 you can cover the tag feature

Variables can go along separately.

Reply:
Lets just keep it all together